packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -352,24 +353,25 @@ export const prepareLocalOrchestrationAccountKit = (
          * @param {{
          *   destination: ChainAddress;
          *   opts?: IBCMsgTransferOptions;
-         *   amount: DenomAmount;
+         *   denomAmount: DenomAmount;
+         *   isPfm: boolean;
          * }} ctx
          */
         onFulfilled(
           [{ transferChannel }, timeoutTimestamp],
-          { opts, amount, destination },
+          { opts, denomAmount, destination, isPfm },
```

Comment:
is renaming the `amount` property a breaking change?

probably doesn't matter yet, but we should keep this sort of thing in mind

Reply:
This change will be part of a new PR, see [comment](https://github.com/Agoric/agoric-sdk/pull/10329#issuecomment-2468528184), but thanks for pointing this out.

This interface is part of .transfer()'s internals, and doesn't get exposed to a consumer. I believe that means it's not a breaking change, but let me know if that's off base. Exo upgrade considerations are something I'm always looking to better understand.

Reply:
The state record gets written to virtual storage. Upgraded exos have to use the same state properties, IIUC.
I think our baggage test captures this. Once the contract is deployed, we're committed to the state in baggage. We can evolve it, but only in certain limited ways.